addons/event/models/event_stage.py

```diff
@@ -17,11 +17,11 @@ class EventStage(models.Model):
         string='End Stage', default=False,
         help='Events will automatically be moved into this stage when they are finished. The event moved into this stage will automatically be set as green.')
     legend_blocked = fields.Char(
-        'Red Kanban Label', default=lambda s: _('Blocked'), translate=True, required=True,
+        'Red Kanban Label', default=lambda s: _('Blocked'), translate=True, prefetch=True, required=True,
```

Comment:
frankly sometimes i also think we should simply drop functionalities like these, best way to get perfs...